src/Connector/Client.php

```diff
@@ -203,4 +210,20 @@ public function addOption($name, $value): void
     {
         $this->options = array_merge_recursive($this->options, [$name => $value]);
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function setUserAgent($name): void
```

Comment:
We may need to change this so the ordering is based on when this method is called. Currently it is alphabetical but we could use a numerical array and just add the value.

Reply:
Reminder to add another user agent in tests on the other side alphabetically to ensure the right order is attained.